admin/acceptance.py

```diff
@@ -58,9 +62,39 @@ def remove_known_host(reactor, hostname):
     return run(reactor, ['ssh-keygen', '-R', hostname])
 
 
-def run_tests(reactor, nodes, control_node, agent_nodes, trial_args):
+def run_client_tests(reactor, node, trial_args):
     """
-    Run the acceptance tests.
+    Run the client acceptance tests.
+
+    :param INode node: The node to run client acceptance tests against.
+    :param list trial_args: Arguments to pass to trial. If not
+        provided, defaults to ``['flocker.acceptance']``.
+
+    :return int: The exit-code of trial.
+    """
+    if not trial_args:
```

Comment:
Oh, `trial_args` is not actually used anywhere in this function. So delete all the noise.